ga_deploy.sh

```diff
@@ -0,0 +1,8 @@
+echo 'Starting deployment...'
+
```

Comment:
Is this file and `ga_tests.sh` required outside the context of github actions ? If not, then lets move these sh files to the workflows folder itself